packages/picker/src/spectrum-picker.css

```diff
@@ -192,13 +192,10 @@ governing permissions and limitations under the License.
     );
 }
 :host([size='m']) {
-    --spectrum-picker-texticon-padding-left: var(
```

Comment:
This causes trouble in Textfield contexts, can you hunt down where this is being removed?

Reply:
Let me see why this is even here in this PR. Maybe I merged main wrong in here.

Reply:
Let me take a look of the latest merge from main. Maybe there was a conflict.

Reply:
Looks like this may be a series of bugs in the Spectrum CSS releases that went out yesterday: https://73874bdaecee6ea6d7a506b37d5d256d--spectrum-web-components.netlify.app/review/#TextfieldStories/Default.png

Reply:
You can see the values removed between https://unpkg.com/browse/@spectrum-css/textfield@3.2.8/dist/index-vars.css and https://unpkg.com/browse/@spectrum-css/textfield@3.2.9/dist/index-vars.css 🤯 @pfulton